core/shared/src/test/scala/fs2/StreamCombinatorsSuite.scala

```diff
@@ -874,6 +874,7 @@ class StreamCombinatorsSuite extends Fs2Suite {
             source.groupWithin(Int.MaxValue, 1.day)
 
           downstream.compile.lastOrError
+            .timeout(downstreamTimeout)
```

Comment:
Why is this timeout necessary? Since its an `executeEmbed` test.

Reply:
it's because if the test fails we get a slightly better error message

```java.util.concurrent.TimeoutException: 7500 milliseconds``` which can be easily associated to `downstreamTimeout`

otherwise we get this value on the diff which looks a bit random

```_1 = 86405500000000 nanoseconds,```

but I'm happy to remove it

Reply:
Got it, that is a nicer error :) thanks!